Makefile

```diff
@@ -212,6 +242,22 @@ $(FOGBENCH_PYTHON_INSTALL_DIR) :
 $(EXTRAS_INSTALL_DIR) :
 	$(MKDIR_PATH) $@
 
+###############################################################################
+####################### DATA INSTALL TARGETS ################################
+###############################################################################
```

Comment:
Data directories should be a product of init rather than install. I understand that we don't have init yet.

Reply:
Yes, but only if it dynamic data. Where we are installing data files that come as part of the package then it is right to build them in the make install. If the package needs them to run then they must be, because the package must be able to run following a make install.

Reply:
In this case, this is a directory that must exist in FOGLAMP_DATA, which has variable location. Installing the directory in the default FOGLAMP_DATA directory is okay as a first step. However, if someone is not using the default FOGLAMP_DATA directory, then the user must manually move the directory contents into the non-default FOGLAMP_DATA  directory.